marsha/settings.py

```diff
@@ -26,10 +26,15 @@ class Base(Configuration):
     """
 
     BASE_DIR = os.path.dirname(__file__)
+    DATA_DIR = os.path.join("/", "data")
```

Comment:
Why not making this setting also configurable _via_ a `Value`?

Reply:
Either this one or static and media but not both?

Reply:
I would say both in case you use `DATA_DIR` for something else not related to statics or media.